src/pages/dashboard/FilterSearchOptions.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { Dispatch, FC, SetStateAction, useEffect, useLayoutEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Colors, DeviceWidths } from '../../themes/mainTheme';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { useTranslation } from 'react-i18next';
+import InstitutionFiltering from './InstitutionFiltering';
+import { QueryObject } from '../../types/search.types';
+
+function useWindowWidth() {
```

Comment:
er denne bare for å sjekke størrelse på skjerm, og endre hva som vises basert på det? i så fall kan man like godt bruke theme og media queries.
Her skjuler vi feks `preview` for små skjermer: https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/src/pages/public_registration/PublicFilesContent.tsx#L36

Reply:
Jeg prøvde det med mediaqeury og display:none, men vi har to forskjellige komponenter som har samme komponent inni. Det førte til at de indre komponentene ble kalt samtidig.